docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week6.md

```diff
@@ -0,0 +1,69 @@
+
+Firstly, congratulations on being half way through Capstone, you should all be very proud of your efforts (I hope you know we are). This also means its time to dive right into your projects.
+
+## OnTrack tasks
+
+Please start working on Task 6.x.
+
+For Task 6.2, this is an update of the company report and project submissions to be added by the 21st to allow time to compile- more information will be coming but:
+
+This will be worked on collaboratively on GitHub by submitting .md PRs to team folder in branch: task6.2/company-report-project-updates
+
+This will include discussion on testing and data strategies and Software Requirement Specification documents, along with a project update
+
+For D/HD aiming students, there are 4 tasks that due on the same day on Sun 24/04/2022 - please be aware of this and plan accordingly
+
+## Company tasks
+
+If you are interested in Leadership, check out the Leadership Conference. The sign up form is here
+
+We know there are still some students who might have questions and don't feel comfortable asking, so we have made a Q&A form and questions will be answered all at once in a FAQ style post.
+
+Please work on the relevant project repositories from the thoth-tech organisation on GitHub. Please do not pull or push to the original Doubtfire or SplashKit repositories - if you have the wrong repo, check out this guide
+
+There will also be a Company wide meeting after Intra-Trimester Break (so be looking out for this). More information will come soon.
+
+You should have seen the new Documentation Repo. If you have any questions, first person to reach out to is Documentation and they can answer your questions.
+
+If the file is relevant companywide and for more than one trimester, it goes in the handbook repo
+
+If the file is relevant to a product or project, it goes in the documentation repo
+
+## What you can expect
+
+Area Leads will be jumping into project meetings this week to catch up on the progress and answer any questions.
+
+From this point on the Area Leads will be taking a step back, first point of call for information is the Products Leads - they are eager to help out.
+
+## What we can expect
+
+By now you should have:
+
+- Received project epics
+- Broken it down into user stories
+- Trello boards filled out in depth with clear communication on what is happening - reviews were given last week
+- Software requirements Specification (SRS)
+- Discussed data requirements
+- Discussed testing requirements
+- Have documented your work and progress
+- Had your first retro
+- Feel confident in what you are doing - if you do not **REACH OUT**
+- Upskilled
+
+We will expect the teams are working on:
+
+- Data and Testing strategy templates filled out this week ready to present in company report
+
+- Beginning work on projects if all up to date (this will be different for all teams)
+
+- Continuing your leadership/cohort contributions (D/HD students)
```

Comment:
**nitpick:** A tiny suggestion

```suggestion
- Data and Testing strategy templates filled out this week ready to present in company report
- Beginning work on projects if all up to date (this will be different for all teams)
- Continuing your leadership/cohort contributions (D/HD students)
```